base/REPLCompletions.jl

```diff
@@ -134,16 +134,68 @@ function complete_path(path::AbstractString, pos)
     return matches, nextind(path, pos - sizeof(prefix) - length(matchall(r" ", prefix))):pos, length(matches) > 0
 end
 
-function complete_methods(input::AbstractString)
-    tokens = split(input, '.')
-    fn = Main
-    for token in tokens
-        sym = symbol(token)
-        isdefined(fn, sym) || return UTF8String[]
-        fn = fn.(sym)
+# Determines whether a string endswiths ',' or '(' when disregarding whitespace_chars
+function ends_c(s::AbstractString)
```

Comment:
This function name is pretty opaque to me. Can we come up with something more descriptive?